src/AngleSharp.Diffing/Core/Comparison.cs

```diff
@@ -25,7 +25,9 @@ namespace AngleSharp.Diffing.Core
         /// <summary>
         /// Gets whether the control and test nodes are of the same type and has the same name.
         /// </summary>
-        public bool AreNodeTypesEqual => Control.Node.NodeType == Test.Node.NodeType && Control.Node.NodeName == Test.Node.NodeName;
+        public bool AreNodeTypesEqual
+            => Control.Node.NodeType == Test.Node.NodeType
```

Comment:
Alternatively, we could also use `.Isi` here (https://github.com/AngleSharp/AngleSharp/blob/devel/src/AngleSharp/Text/StringExtensions.cs#L408). This already compares using case-insensitive matching.

Reply:
That's a neat method. But I think I will stick with the standard one from the framework to avoid surprising myself and other devs looking at the code since I am not using the `Is` and `Isi` methods elsewhere. Consistency is king in my book.